Automation/deployer.py

```diff
@@ -0,0 +1,75 @@
+# from https://github.com/Azure-Samples/resource-manager-python-template-deployment
+
+"""A deployer class to deploy a template on Azure"""
```

Comment:
I didn't end up using this, did you? I think we can delete

Reply:
Nope, I just kept it since I thought you wanted to implement it. I don't think we need it, I will delete. Thanks.